msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -543,7 +565,7 @@ async def on_teams_members_added(
 
 ### Team members removed
```

Comment:
We will need to update this section as well to "Members removed"

Let's format this section into bullets, similar to what we did for "members added" section so that the sections are consistent

> A member removed event is sent to your bot in the following scenarios:
>1. When the bot, itself, is uninstalled and removed from a conversation
>2. When a user is removed from a conversation where the bot is installed
>
>The member removed activity will have `eventType` set to `teamMemberRemoved` when the event is sent from a team context.
>
>To determine if the member removed was the bot itself or a user, check the `Activity` object of the `turnContext`. If the `MembersRemoved` list contains an object where `id` is the same as the `id` field of the `Recipient` object, then the member added is the bot, else it's a user. The bot's `id` is formatted as `28:<MicrosoftAppId>`.